src/frontend/js/components/DashboardCourses/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { useOrders } from 'hooks/useOrders';
+import { DashboardItemOrder } from 'components/DashboardItem/Order/DashboardItemOrder';
+import { OrderState } from 'types/Joanie';
+import { Spinner } from 'components/Spinner';
+
+const messages = defineMessages({
+  loading: {
+    defaultMessage: 'Loading ...',
+    description: 'Message displayed while loading orders and enrollments',
+    id: 'components.DashboardCourses.loading',
+  },
+});
+
+export const DashboardCourses = () => {
+  const orders = useOrders();
```

Comment:
You could filter orders by state 🤔 but it creates a new query 😕

Reply:
My guess is that it is cheaper to do this quick filtering than to register a new query in this situation 🤔